source/includes/_reference.md

```diff
@@ -131,23 +131,16 @@ def enable_scoutprof?; true; end
 
 ### Free Trial
 
-We offer a no-risk, free 14-day trial. Delete your monitored apps before the trial concludes and you don't pay.
+We offer a no risk, fully featured, free trial. Enter a credit or debit card anytime to continue using Scout APM after the end of your trial.
 
 ### Billing Date
 
 Your first bill is 30 days after your signup date.
```

Comment:
Is this true?

Reply:
True-ish? The first bill is 30 days after an invoice has been generated, not 30 days after signing up for Scout. It could also be 30 days after signing up for a subscription. Either way, "signup date" could be confusing.  I'm assuming that an invoice is generated as soon as a payment method is entered.